core/controllers/domain_objects_validator.py

```diff
@@ -105,6 +151,9 @@ def validate_email_dashboard_data(data):
 
     Args:
         data: dict. Data that needs to be validated.
```

Comment:
Try not to just name it `data`, that's too vague.

Reply:
Updated.